specificatie/naam.yaml

```diff
@@ -0,0 +1,173 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Aanhef:
+      type: string
+      description: |
+        Kun je gebruiken als aanhef in een brief gericht aan persoon.
+      example: "Hoogwelgeboren heer"
+    Aanschrijfwijze:
```

Comment:
@MelvLee hier definieer je een object in een property. Voorheen vonden we dat niet oké, omdat dan bij code genereren de generator een class-naam verzint. Hier geen probleem?

Reply:
Dit begrijp ik niet Aanschrijfwijze is inderdaad een type/object geworden en volgens mij ook als zodanig gedefinieerden niet als inline type/object.

Reply:
hem ik heb denk ik scheel gekeken